kernel/aster-nix/src/fs/utils/inode.rs

```diff
@@ -330,7 +330,11 @@ pub trait Inode: Any + Sync + Send {
         Err(Error::new(Errno::EISDIR))
     }
 
-    fn sync(&self) -> Result<()> {
+    fn sync_all(&self) -> Result<()> {
+        Ok(())
+    }
+
+    fn sync_data(&self) -> Result<()> {
```

Comment:
These two empty implementations can be removed since the trait has provided default implementations.

Reply:
That's right. Thank you for your reminder!